packages/valory/skills/market_creation_manager_abci/rounds.py

```diff
@@ -771,7 +771,7 @@ class MarketCreationManagerAbciApp(AbciApp[Event]):
     transition_function: AbciAppTransitionFunction = {
         PostTransactionRound: {
             Event.DONE: CloseMarketsRound,
-            Event.ERROR: PostTransactionRound,
+            Event.ERROR: CloseMarketsRound,
```

Comment:
Why would we end up in a situation where we cannot execute the logic in the PostTransaction cannot be executed properly?
Seems like the issue is elsewhere. Eitherway, it makes sense that we transition outside this round.

Reply:
This happened on #82, the service remained stuck there undefinitely (I had to restart it).